docs/examples/test_ansatz.py

```diff
@@ -0,0 +1,28 @@
+from typing import Optional
+
+import numpy as np
+from orquestra.quantum.circuits import RZ, Circuit
+from orquestra.vqa.api import Ansatz
+from sympy import symbols
+
+
+class TestAnsatz(Ansatz):
```

Comment:
I think it would be better to rename it from `TestAnsatz` to `MockAnsatz` or `ExampleAnsatz` even?

Reply:
Renamed to `MockAnsatz` to keep constistency with our other mocks.